twisted/plugins/fedmsg_verify_missing_plugin.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of fedmsg_migration_tools.
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Comment:
A short docblock for this would be good. I'm a *little* familiar with twisted plugins, but having a quick description and links here would be helpful. Is this to work with the twistd CLI? Do we need that since we've already got a CLI?